plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -10,10 +10,15 @@ class Release < ActiveRecord::Base
 
     validates :release_group, presence: true
     validates :deploy_group, presence: true
-    validates :status, inclusion: VALID_STATUSES
+    validates :status, inclusion: STATUSES
 
     after_initialize :set_default_status, on: :create
 
+    def release_is_live
+      self.status = 'live'
+      self.deploy_finished_at = Time.now
```

Comment:
This is generally done with activerecord models, but I'm not sure of the reasoning behind it. I think it's because activerecord defines a operator= method override for all columns in table?